phpBB/styles/prosilver/template/navbar_header.html

```diff
@@ -186,8 +186,10 @@
 				<span class="crumb" {$MICRODATA}><a href="{U_INDEX}" itemprop="url" accesskey="h" data-navbar-reference="index"><!-- IF not U_SITE_HOME --><i class="icon fa-home fa-fw"></i><!-- ENDIF --><span itemprop="title">{L_INDEX}</span></a></span>
 
 			<!-- BEGIN navlinks -->
+				{{ set NAVINK_NAME = navlinks.BREADCRUMB_NAME || navlinks.FORUM_NAME }}
+				{{ set NAVINK_LINK = navlinks.U_BREADCRUMB || navlinks.U_VIEW_FORUM }}
```

Comment:
May I ask why `NAVINK_*` isn't `NAVLINK_*`? Out of curiosity...

Reply:
Typo. 😅